packages/webapp/src/components/ReleaseBadge/styles.module.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: 400px;
+  height: fit-content;
+
+  border-radius: 8px;
+  background: #ffdad9;
+  box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+
+  position: fixed;
+  bottom: 32px;
+  z-index: 1400; // highest existing z-index is 1302
```

Comment:
z-index wars continue.. haha 

Makes me think there is probably a MUI component that handles all this functionality without making a custom component. Something like snackbar, alert, or popover.

Reply:
We could also refactor it into a more sensible structure without MUI's help 😂  I think I've read a blog post (did you send it??!) against the random z-index situation we have gotten ourselves into.

Reply:
yes haha z-index bad, me no likey.  -- I think the 1302 is to override MUI styles too. This was the article: https://www.codegram.com/blog/stop-ab-using-z-index/

MUI is pretty nice though -- an info Alert makes a lot of sense to me -- the blue color is so nice too ;) https://mui.com/material-ui/react-alert/